GUI/DeusEx/Classes/DXRComputerScreenHack.uc

```diff
@@ -14,11 +15,15 @@ var int biocellCount;
 function Tick(float deltaTime)
 {
     local Human p;
+    local DXRando dxr;
+
+    dxr = class'DXRando'.default.dxr;
+
     if (bHacking)
     {
         p = Human(winTerm.compOwner.Owner);
         if( p == None ) p = Human(Player);// ATMs don't set the Owner
-        if( p != None && !p.bZeroRando ) {
+        if( p != None && !dxr.flags.IsZeroRando() ) {
```

Comment:
`p.bZeroRando = IsZeroRandoPure();`

I'm changing these to appropriately use `dxr.flags.IsZeroRandoPure()`

the bool name is probably bad, but I want to refactor that anyways to use global variables stored in `class'DXRFlags'.default.bZeroRandoPure`, after v3.3 I guess